docs_user/modules/proc_pulling-configuration-from-a-tripleo-deployment.adoc

```diff
@@ -60,59 +66,41 @@ services:
       - /etc/keystone/keystone.conf
       - /etc/keystone/logging.conf
 ----
++
 Repeat this step for each {OpenStackShort} service that you want to disable or enable.
 
-. If you are using non-containerized services, such as the `ovs-external-ids`, os-diff can pull configuration or command output:
+. If you use non-containerized services, such as the `ovs-external-ids`, pull the configuration or the command output. For example:
 +
 ----
 services:
   ovs_external_ids:
-    hosts:
+    hosts: <1>
       - standalone
-    service_command: "ovs-vsctl list Open_vSwitch . | grep external_ids | awk -F ': ' '{ print $2; }'"
-    cat_output: true
+    service_command: "ovs-vsctl list Open_vSwitch . | grep external_ids | awk -F ': ' '{ print $2; }'" <2>
+    cat_output: true <3>
     path:
       - ovs_external_ids.json
-    config_mapping:
-      ovn-bridge-mappings: edpm_ovn_bridge_mappings
+    config_mapping: <4>
+      ovn-bridge-mappings: edpm_ovn_bridge_mappings: ["datacentre:br-ex"]
       ovn-bridge: edpm_ovn_bridge
```

Comment:
@ralonsoh You submitted a bug about this change. Does this update look right? 
https://issues.redhat.com/browse/OSPRH-9087